tokio/src/time/driver/handle.rs

```diff
@@ -47,7 +47,7 @@ cfg_rt! {
         /// panicking.
         pub(crate) fn current() -> Self {
             crate::runtime::context::time_handle()
-                .expect("there is no timer running, must be called from the context of Tokio runtime")
+                .expect("a Tokio 1.x context was found, but timers are disabled")
```

Comment:
```suggestion
                .expect("A Tokio 1.x context was found, but timers are disabled. Call `enable_time` on the runtime builder to enable timers.")
```

Reply:
@Darksonn Why do you prefer capitalizing panic message here, since it's generally not otherwise (f.ex. `util::error::context_missing_error`)?